bigtable-test/bigtable-emulator-maven-plugin/README.md

```diff
@@ -66,4 +67,47 @@ Usage:
       Connection connection = BigtableConfiguration.connect("fakeproject", "fakeinstace");
    ```
 - If `maven.test.skip` is set to true the emulator will not start. The emulator can also be controlled directly 
-  by setting the boolean property `bigtable.emulator.skip`.
\ No newline at end of file
+  by setting the boolean property `bigtable.emulator.skip`.
+
+- While using Docker, You can use container's `host` and `port` details for emulator:
```

Comment:
While..., you can use the container's...(You should not be capilatlized and add the)